src/Body.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function Body() {
```

Comment:
Try to break this into different component with more descriptive names and uses. For example Body will not tell you much when you have multiple pages.

Reply:
`Body' component has been removed as I believe that it didn't need to exist as a separate component.